doc/administrate.rst

```diff
@@ -139,8 +139,7 @@ is installed on the server, fedmsg messages are published for these events.
 Generate cache
 --------------
 
-To decrease the weight of the page where all the candidates of an election
-are shown, nuancier generates thumbnails.
+In order to reduce the size(and hence the loading time) of the page where all the candidates of an election are shown, nuancier generates thumbnails.
```

Comment:
ok for the change, but it is more than 80 chars, so it needs a ``\n`` after ``where`` I guess